source/content/glossar/bibliothek/geocat-mappings/distribution-rights.rst

```diff
@@ -1,8 +1,8 @@
 :DCAT: :ref:`dct:rights <dcat-distribution-rights>`
 :XPATH: ``//gmd:resourceConstraints//gmd:otherConstraints``
 :Mapping of values: For the mapping of the rights see the vocabulary below:
-          :download:`RIGHTS-RDF <../../../_static/vocabulary/rights.rdf>`,
-          :download:`RIGHTS-Turtle <../../../_static/vocabulary/rights.ttl>`,
+          :download:`RIGHTS-RDF </_static/vocabulary/rights.rdf>`,
+          :download:`RIGHTS-Turtle </_static/vocabulary/rights.ttl>`,
```

Comment:
This is very strange behaviour. The file we want to link to is at (locally, for me) `Projects/opendata/ogdch-handbook-site/docs/source/_static/vocabulary/rights.rdf`

With `../../../_static/vocabulary/rights.rdf`, Sphinx looked for a file at `Projects/opendata/ogdch-handbook-site/docs/source/content/_static/vocabulary/rights.rdf` and didn't find it because it's not there.

With `../../../../_static/vocabulary/rights.rdf`, adding one more `../`, which should have been correct, Sphinx looked for a file at `Projects/opendata/_static/vocabulary/rights.rdf`, far outside where it should have been.

Specifying an absolute file path (from `Projects/opendata/ogdch-handbook-site/docs/source/`) works correctly. I don't know if we deliberately used a relative path here - it works as expected elsewhere, e.g. https://github.com/opendata-swiss/ogdch-handbook/blob/develop/source/content/glossar/bibliothek/dcat-ap-ch.rst#L118.

Reply:
I just checked and it's not just that we get a warning - the links specified with relative paths don't work, see https://ogdch-new-handbook.clients.liip.ch/de/content/glossar/bibliothek/geocat-mappings/distribution-rights.html. They do work locally with these changes. I am baffled but suggest we just use the absolute paths as here.